pubspec.yaml

```diff
@@ -1,5 +1,5 @@
 name: sass_migrator
-version: 1.0.0
+version: 1.0.1
```

Comment:
The pattern we used on the Dart team, which I still use on Dart Sass, is to use the `-dev` suffix for versions that are still in-development and not intended to be released. Notionally this means that the only Git revisions that have non-`-dev` versions are exactly the same as the tagged and published versions. This also helps when collaborating with other people, because they know without checking published venues that if the version is `-dev` they can add new features without bumping the version further but if it's not they need to bump it.

So `1.0.1` would make sense if you were planning on tagging and publishing immediately, but otherwise you'd want to write `1.0.1-dev`.

Reply:
Oh, okay. I'll do that in the future. In this case, I'd like to fix #114 (PR incoming) before publishing `1.0.1`.